src/org/usfirst/frc/team694/robot/commands/DrivetrainLineSensorPlatformZoneCommand.java

```diff
@@ -0,0 +1,15 @@
+package org.usfirst.frc.team694.robot.commands;
+
+import org.usfirst.frc.team694.robot.FieldMapInterface;
+import org.usfirst.frc.team694.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+public class DrivetrainLineSensorPlatformZoneCommand extends CommandGroup {
+    public static FieldMapInterface Quad = Robot.getRobotQuadrant();
```

Comment:
Quad is a variable, follow variable conventions.
The variable shouldn't be static and public